docs/building-a-bento-amp-extension.md

```diff
@@ -551,23 +590,23 @@ exports.extensionBundles = [
 ];
 ```
 
-Note, if you are providing a version upgrade (pre-existing 0.1 to Bento 1.0, for example), it is important to note that the latest version is the 0.1 until the 1.0 version is no longer experimental and fully launched. The following is the most straight-forward way to specify this:
+The entry for your component must have `options.wrapper = "bento"`, and may optionally include `options.npm = true` to opt the Preact component portion for automated npm publishing on AMP's release schedule.
 
 ```javascript
 exports.extensionBundles = [
 ...
-  {name: 'amp-my-element', version: ['0.1', '1.0'], latestVersion: '0.1', options: {hasCss: true}},
+  {name: 'amp-my-element', version: '1.0', latestVersion: '0.1', options: {npm: true, wrapper: "bento"}},
```

Comment:
This file is JSON now. We should format this code and the below snippet as well for length.

```json
{
  "name": "amp-my-element",
  "version": "1.0",
  "latestVersion": "0.1",
  "options": {
    "npm": true,
    "wrapper": "bento"
  }
}
```

Reply:
Done, thanks